remote-process/main.cpp

```diff
@@ -33,15 +33,14 @@
 #include <stdlib.h>
 #include "RequestMessage.h"
 #include "AnswerMessage.h"
-#include "ConnectionSocket.h"
 
 using namespace std;
 
-bool sendAndDisplayCommand(CConnectionSocket &connectionSocket, CRequestMessage &requestMessage)
+bool sendAndDisplayCommand(asio::ip::tcp::socket &socket, CRequestMessage &requestMessage)
```

Comment:
Can we have more generic socket object here ?
At least no restriction on domain